modules/etcd-defrag.adoc

```diff
@@ -6,10 +6,43 @@
 [id="etcd-defrag_{context}"]
 = Defragmenting etcd data
 
-Manual defragmentation must be performed periodically to reclaim disk space after etcd history compaction and other events cause disk fragmentation.
+Defragmentation of etcd data reclaims disk space after etcd history compaction and other events  cause disk fragmentation.
 
 History compaction is performed automatically every five minutes and leaves gaps in the back-end database. This fragmented space is available for use by etcd, but is not available to the host file system. You must defragment etcd to make this space available to the host file system.
 
+Defragmentation occurs using one of the following options:
+* Automatic
+* Manual
+
+[NOTE]
+====
+Automatic defragmentation is preferred because the etcd operator uses cluster information to determine the most efficient operation for the user.
+====
+
+.Automatic defragmentation
+
+Automatic defragmentation is initiated by the etcd operator and occurs during set intervals. This interval is determined by the etcd operator on a time-based basis and no manual intervention is needed.
```

Comment:
Passive

Also, we always capitalize "Operator" for historical reasons. :cry: 

```suggestion
The etcd Operator perform disk defragmentation without intervention during set intervals. This interval is determined by the etcd operator on a time-based basis.
```

I'm not sure about the second sentence. Is it possible to be more specific? Are the intervals at all something we can configure? Is defragmentation based on disk usage patterns? 

If no to all of that, could go even shorter: "The etcd Operator automatically defragments disks. No manual intervention is needed."